Protos/V1/submitter_service.proto

```diff
@@ -149,4 +163,28 @@ message WaitRequest {
 	TaskFilter filter = 1;
 	bool stop_on_first_task_error = 2;
 	bool stop_on_first_task_cancellation = 3;
-}
\ No newline at end of file
+}
+
+/**
+ * Request to watch result states
+ * It contains the list of result ids you want to watch
+ *   and some options to filter out some events
+ */
+message WatchResultRequest {
+  // list of statuses to check results against for the initial fetch
+  repeated ResultStatus.ResultStatus fetch_statuses = 1;
+  // list of statuses to check results against for the watch
+  repeated ResultStatus.ResultStatus watch_statuses = 2;
+  // result ids to fetch/watch
+  repeated string result_ids = 3;
+}
+
+/**
+ * List of Result statuses
+ */
+message WatchResultStream {
+  // Status of the results
+  ResultStatus.ResultStatus status = 1;
+  // List of result ids that triggered the event
```

Comment:
L'idee c'est de retourner un message par statut voire plusieurs s'il y a trop d'ids d'un coup ? Je pense qu'on pourrait ajouter un peu de détails dans la doc de ce message. ça a été fait pour le message précédent.

Reply:
Je ne parlerais pas de chunking du tout ! Il n'y a pas de différence entre envoyer 2x plein d'ids et 1x plein d'Id et juste après plein d'Id qui viennent de changer de statut.

Reply:
maintenant explicité

Reply:
My proposal : "Result Ids are grouped by status. One message contains result Ids that have the same status."